### ISSUE TITLE
You call document ready twice

### ISSUE THREAD

**Issue by epascarello** [LiranCohen/stickUp#4]
jQuery(function) is document.ready!

http://api.jquery.com/jQuery/#jQuery3

```
jQuery(  <-- this is shortcut for document.ready
function($) {        
        $(document).ready(function(){   <-- this is document.ready inside of a document.ready
```

**Comment by Sparticuz** [comment]
I was coming to say the same thing

**Comment by kaynenh** [comment]
Yes. This definitely caused console errors for me. I removed the inner document.ready and everything worked smoothly.

**Comment by jherencia** [comment]
Same problem here.

**Comment by dynamicdan** [comment]
This stopped it working for me too. Lost at least 15 mins :-(

**Comment by danalmeida** [comment]
Yea, this was causing the plugin to fail for me as well. I kept getting an error until i removed the `$(document).ready()` wrapper on line 4

**Comment by tahitiangabriel** [comment]
Same for me too. Did lost a couple of hours on this!
The project seems not to be maintain anymore...
